kotsadm/pkg/supportbundle/redactions.go

```diff
@@ -0,0 +1,51 @@
+package supportbundle
+
+import (
+	"database/sql"
+	"encoding/json"
+	"fmt"
+
+	"github.com/pkg/errors"
+	"github.com/replicatedhq/kots/kotsadm/pkg/persistence"
+	"github.com/replicatedhq/troubleshoot/pkg/redact"
+)
+
+func GetRedactions(bundleID string) (redact.RedactionList, error) {
+	db := persistence.MustGetPGSession()
+	q := `select redact_report from supportbundle where id = $1`
+
+	var redactString sql.NullString
+	row := db.QueryRow(q, bundleID)
+	err := row.Scan(&redactString)
+	if err != nil {
+		return redact.RedactionList{}, errors.Wrap(err, "select redact_report")
+	}
+
+	if !redactString.Valid {
```

Comment:
Better check `redactString.String == ""` because that column allows empty string values